website_supplier_list/views/website_supplier.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+<!-- Layout add nav and footer -->
+<template id="footer_supplier_page" inherit_id="website.footer_default" name="Footer Supplier Link">
+    <xpath expr="//div[@id='info']/ul" position="inside">
+        <li><a href="/suppliers">Our Suppliers</a></li>
+    </xpath>
+</template>
+
+<template id="index" name="Our Suppliers">
+    <t t-call="website.layout">
+        <div id="wrap">
+            <div class="oe_structure">
+                <section data-snippet-id="title">
+                    <h1 class="text-center">
+                        Our Suppliers
+                    </h1><h2 class="text-center text-muted">
+                        Trusted by millions worldwide
+                    </h2>
+                </section>
+            </div>
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-3 mb32" id="ref_left_column">
+                    </div>
+                    <div class="col-md-8 col-md-offset-1" id="ref_content">
+                        <div class='navbar mb0'>
+                            <t t-call="website.pager">
+                               <t t-set="classname" t-value="'pull-left'"/>
+                            </t>
+                            <form action="" method="get" class="navbar-search pull-right pagination form-inline">
```

Comment:
This form lacks a submit button. Idea: http://getbootstrap.com/components/#input-groups-buttons